addons/l10n_din5008_sale/models/sale.py

```diff
@@ -43,11 +43,26 @@ def _compute_l10n_din5008_document_title(self):
     def _compute_l10n_din5008_addresses(self):
         for record in self:
             record.l10n_din5008_addresses = data = []
-            if record.partner_shipping_id == record.partner_invoice_id:
-                data.append((_("Invoicing and Shipping Address:"), record.partner_shipping_id))
-            else:
-                data.append((_("Shipping Address:"), record.partner_shipping_id))
-                data.append((_("Invoicing Address:"), record.partner_invoice_id))
+            commercial_partner = record.partner_id.commercial_partner_id
```

Comment:
Maybe it's better to do a helper function since we have the same thing in account

Reply:
personally I don't think it needs to be a helper, I like the fact that you can see the logic in the place it is applied, but that's just my opinion.

Reply:
IMHO, one repetition is ok, two is a bit disturbing but ok, but after we need a helper. 
So for that PR, let's keep it like this.